src/lib/auth.ts

```diff
@@ -1,16 +1,31 @@
 import { Lucia } from "lucia";
-import { GitHub } from "arctic";
+import { GitHub, Google } from "arctic";
 import { LibSQLAdapter } from "@lucia-auth/adapter-sqlite";
 import { connection } from "@/db";
 import { env } from "@/env";
 import { cache } from "react";
 import { cookies, headers } from "next/headers";
 
+export const PROVIDER = {
+  GITHUB: "GITHUB",
+  GOOGLE: "GOOGLE",
+} as const;
+
+type ObjectValues<T> = T[keyof T];

```

Comment:
Maybe we can put this in another file?
e.g. `@/lib/type-util`
This seems to be reusable later

Reply:
yeah we can move it definition file (d.ts) or global but currently only use it here